src/commands/slack-notify-waiting-for-approval.yml

```diff
@@ -0,0 +1,89 @@
+description: |
+  Notifies the current workflow owner that their workflow is pending approval to proceed.
+  Depends on jq being present.
+
+  Setup:
+    1. Create Coda document with a table containing columns `circleci alias` (ex: github username) and `email`, and populate this table with information for each github user.
+    3. Find the docId, tableId, and columnIds for the circleci user alias and email columnIds.
+    4. Create a Coda API token for this document at https://coda.io/account - limit the token to read access to the target table.
+    5. Setup a Slack Bot account with scopes `users:read`, `users:read.email`, and `chat:write`.
+    6. Configure orb based on required args
+parameters:
+  CODA_USER_ROSTER_TABLE_URL:
+    description: |
+      Fully qualified API URL to a table containing CIRCLECI_USERNAMEs to email aliases.    Must be of the form
+      https://coda.io/apis/v1/docs/<DOCID>/tables/<TABLEID>/rows.
+    type: string
+    default: https://staging.coda.io/apis/v1/docs/s2i6oFeghW/tables/grid-QGyaiXZDwu/rows
+  CODA_API_TOKEN:
+    description: |
+      Env var of a token granted read access to the CODA_USER_ROSTER_TABLE_URL document.
+    type: string
+    default: ${STAGING_CODA_TOKEN}
+  CODA_CIRCLECI_USER_NAME_COL:
+    description: |
+      Coda columnId of the column storing the CircleCI username.
+    type: string
+    default: c-6ni4kHGNwE
+  CODA_CIRCLECI_USER_ALIAS_COL:
+    description: |
+      Coda columnId of the column storing the user alias (when using EMAIL_DOMAIN), or fully qualified email.
+    type: string
+    default: c-26If9Zttyp
+  EMAIL_DOMAIN:
+    description: |
+      Optional email domain for users within the workspace.   Must be specified if user aliases are not fully qualified.
+    type: string
+    default: coda.io
+  SLACK_BOT_TOKEN:
+    type: string
+    description: |
+      Token used by Slack bot application.   Must have scopes `users:read`, `users:read.email`, and `chat:write`.
+    default: ${PUSH_REMINDER_BOT_TOKEN}
+  vcs:
+    description: |
+      Defaults to "gh", but allows other version control systems.
+    type: string
+    default: gh
+steps:
+- run:
+    name: Fetch User Information from Look Up Table
+    command: <<include(scripts/fetch_user_handles.sh)>>
+- when:
+    condition: $SLACK_USER_ID
+    steps:
+    - run:
+        name: Send Slack Message 
+        command: |
+          set -eo pipefail
+          curl -X POST -H "Authorization: Bearer $PUSH_REMINDER_BOT_TOKEN" \
+            -H "Content-Type: application/json" -d \
+            "{ \
+              \"channel\": \"${SLACK_USER_ID}\", \
+              \"attachments\": [ \
+                { \
+                  \"text\": \"Pending Approval for ${CIRCLE_PROJECT_REPONAME} push (${CIRCLE_USERNAME})\", \
```

Comment:
Nit: maybe remove "push"?  When we first wrote this it was used for config pushes but is now used as a general approval flow.